src/app/models/building_color_bucket_calculator.js

```diff
@@ -106,10 +121,10 @@ define([
 
     let scale;
     if (this.thresholds) {
-      scale = d3.scale.threshold().domain(this.thresholds).range(stops);
+      scale = d3.scaleThreshold().domain(this.thresholds).range(stops);
```

Comment:
Same here. Only minor changes to this file to update d3 V3 syntax to d3 V4 since it's better supported/documented.